app/views/application/_text_area.html.erb

```diff
@@ -1,5 +1,8 @@
 <div class="<%= local_assigns[:container_classes]%>">
   <%= form.label attribute %>
+  <% if local_assigns[:label_hint] %>
+    <small><%= local_assigns[:label_hint] %></small>
+  <% end %>
```

Comment:
Thank you! I was thinking ti would be nice to have a hint! But I didn't add it at the moment.